CHANGELOG.md

```diff
@@ -6,7 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic
 Versioning](https://semver.org/spec/v2.0.0.html).
 
-<!-- ## [Unreleased] -->
+## [Unreleased]
+
+### Changed
+
+- The minimum Node version is now `14.0.0` instead of `16.0.0`.
```

Comment:
```suggestion
- The minimum Node version is now `14.14.0` instead of `16.0.0`.
```

Reply:
oops, i was a little late on this. maybe it can be fixed when making a PR for the release.

Reply:
Oops, sorry I missed this. Fixed in https://github.com/google/wireit/pull/161.